raddb/mods-available/mschap

```diff
@@ -336,7 +336,7 @@ mschap {
 		#  TIP: We give both examples here, but *only one should be used*.
 		#
 #		local_cpw = "%exec(/path/to/script %mschap(User-Name) %{MS-CHAP-New-Password.Cleartext})"
-#		local_cpw = "%sql(UPDATE radcheck set value='%{MS-CHAP-New-NT-Password}' where username='%{User-Name}' and attribute='Password.NT'}"
+#		local_cpw = "%sql(UPDATE radcheck set value='%{MS-CHAP-New-NT-Password}' where username='%{User-Name}' and attribute='Password.NT')"
```

Comment:
The argument parser will get upset over spaces in the arguments.  This should be:
```
local_cpw = %sql("UPDATE radcheck set value='%{MS-CHAP-New-NT-Password}' where username='%{User-Name}' and attribute='Password.NT'")
```

i.e. the outer quotes aren't necessary.  But the inner quotes are.  The `"UPDATE... where ..."` statement is one string, and has to be passed to to the `%sql()` function as one string.